chia/cmds/sim.py

```diff
@@ -0,0 +1,216 @@
+from __future__ import annotations
+
+import asyncio
+from pathlib import Path
+from typing import Any, Optional
+
+import click
+
+from chia.cmds.sim_funcs import async_config_wizard, farm_blocks, print_status, revert_block_height, set_auto_farm
+from chia.util.default_root import SIMULATOR_ROOT_PATH
+
+
+@click.group("sim", short_help="Configure and make requests to a Chia Simulator Full Node")
+@click.option(
+    "-p",
+    "--rpc-port",
+    help=(
+        "Set the port where the Simulator is hosting the RPC interface. "
+        "See the rpc_port under full_node in config.yaml"
+    ),
+    type=int,
+    default=None,
+)
+@click.option(
+    "--root-path", default=SIMULATOR_ROOT_PATH, help="Simulator root folder.", type=click.Path(), show_default=True
+)
+@click.option(
+    "-n",
+    "--simulator_name",

```

Comment:
Did we want to clean up all the `_` or no?  (various other cases)

Reply:
meant to, srry